pkg/cmd/component/component/component.go

```diff
@@ -0,0 +1,32 @@
+package component
+
+import (
+	"fmt"
+
+	"github.com/redhat-developer/kam/pkg/cmd/utility"
+	"github.com/spf13/cobra"
+)
+
+//  CompRecommendedCommandName is the recommended Component command name.
+const CompRecommendedCommandName = "component"
+
+// NewCmdComp create a new environment command
+func NewCmdComp(name, fullName string) *cobra.Command {
```

Comment:
The non-interactive mode isn't quite working.  If you run `kam component` it should give you the help.  Compare it with `kam environment`

Reply:
I will look into this.

Reply:
Ok looks good